pyuvdata/uvdata/uvdata.py

```diff
@@ -772,8 +773,8 @@ def __init__(self):
         super(UVData, self).__init__()
 
     @staticmethod
-    def new(**kwargs):
-        """Stand-in docstring for new method."""
+    @combine_docstrings(new_uvdata)
+    def new(**kwargs):  # noqa: D102
```

Comment:
something's going wrong with this docstring's formatting currently in the RTD build. Starting part way through the `antenna_positions` parameter, the formatting goes to putting all the rest of the text into a paragraph that is out-dented so that it is no longer part of the parameters section and is not a bulleted list.